docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-merchant-feature-integration.md

```diff
@@ -37,28 +37,3 @@ Make sure that the following modules have been installed:
 | MerchantProfileMerchantPortalGui | vendor/spryker/merchant-profile-merchant-portal-gui |
 
 {% endinfo_block %}
-
-### 2) Set up transfer objects
-
-Generate transfer changes:
-
-```bash
-console transfer:generate
```

Comment:
lets leave this without verification

Reply:
Well, I don't see any problem here as all transfers and its props that used here are set in `Merchant Feature`.

Reply:
just in case new property will appear or we missed something or some transfers will be removed from dependable features - in this cause this IG will be still actual

Reply:
Ok. Added.